nixos/modules/services/networking/nebula.nix

```diff
@@ -157,6 +172,10 @@ in
             am_lighthouse = netCfg.isLighthouse;
             hosts = netCfg.lighthouses;
           };
+          relay = {
+            am_relay = netCfg.isRelay;
+            relays = netCfg.relays;
+          };
```

Comment:
Since we already have `services.nebula.networks.<name>.settings`, is this necessary? I don't think the relay configs are a required option to define, so creating an explicit option for them may be superfluous.

Reply:
I basically did this to have it be parallel to lighthouse config since it's a very similar format.